src/main/java/me/figo/internal/StartProviderSyncRequest.java

```diff
@@ -25,6 +27,12 @@ public StartProviderSyncRequest(String state, String redirect_uri, boolean disab
     @Expose
     public String state;
 
+    /**
+     * Defines the scope of the synchronization.
+     */
+    @Expose
+    public List<String> scope;
```

Comment:
Maybe it would be a good idea to create an Enum for the scope. Currently only "ACCOUNTS" and "TRANSACTIONS" are valid values.

Reply:
...so it would be a list of Enums (just to clarify)